src/redux/news/news.reducer.js

```diff
@@ -11,7 +11,7 @@ const list = (state = initialState, action) => {
         ...state,
         list: action.payload
       };
-    case NEWS_LOADED:
+    case GET_NEWS:
```

Comment:
You don't need to handle this action type here, it is useless

Reply:
deleted